### PR TITLE
Fixes for ubuntu support

### PR DESCRIPTION
**Pull request by mcrauwel**
move fixes to seperate PR as request in #19.

### CODE REVIEW

File: manifests/params.pp [hunk 1]
```diff
@@ -24,12 +24,18 @@
...
   case $::operatingsystem {
-    'Debian', 'Ubuntu': {
+    'Debian': {
       $admin_listen_socket = '/tmp/proxysql_admin.sock'
       $package_provider    = 'dpkg'
       $sys_owner           = 'root'
       $sys_group           = 'root'
     }
+    'Ubuntu': {
+      $admin_listen_socket = '/tmp/proxysql_admin.sock'
```
Comment (bastelfreak) [inline manifests/params.pp]: is it a good idea to put a socket to /tmp?

File: templates/my.cnf.erb [hunk 2]
```diff
@@ -2,8 +2,11 @@
 prompt = 'Admin> '
 
 [client]
+<% if scope.lookupvar('::proxysql::admin_listen_socket') != '' -%>
```
Comment (bastelfreak) [inline templates/my.cnf.erb]: can we work with undef so this works?

```erb
<% if scope.lookupvar('::proxysql::admin_listen_socket') -%>
```

File: manifests/params.pp (hunk 1 above)
Comment (mcrauwel) [inline manifests/params.pp]: /tmp is the upstream ProxySQL default...

File: templates/my.cnf.erb (hunk 2 above)
Comment (mcrauwel) [inline templates/my.cnf.erb]: I've been working on this last night but I couldn't get it to work with `undef`. 
The parameter is currently defined as:
```
String $admin_listen_socket = $::proxysql::params::admin_listen_socket
```
The default in params is: 
```
$admin_listen_socket = '/tmp/proxysql_admin.sock'
```
When I change the definition to: 
```
Variant[String, Undef] $admin_listen_socket = $::proxysql::params::admin_listen_socket
```
Then I can do:
```
class { '::proxysql': 
 admin_listen_socket => undef,
}
```
but that will go on and use the default `/tmp/proxysql_admin.sock`.

Either I leave it like this or I remove the if and the entire else case and we don't support an empty `$admin_listen_sock`...

Comment (ffrank) [inline templates/my.cnf.erb]: I vote empty string is fine then.

File: manifests/params.pp (hunk 1 above)
Comment (ffrank) [inline manifests/params.pp]: Then it's fine by me FWIW.

**Comment by mcrauwel** [comment]
@bastelfreak could you try to find some time to finish your review here? thnx ;)

**Comment by bastelfreak** [comment]
ah sure!

**Pull request merged**